### PR TITLE
Add scBio R package

### PR DESCRIPTION
**Pull request by PelzKo**
Added the R package scBio (https://github.com/amitfrish/scBio)
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/r-scbio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{% set version = "0.1.4" %}
+
+package:
+  name: r-scbio
+  version: {{ version }}
+
+source:
+  url:
+    - https://github.com/amitfrish/scBio/releases/download/{{ version }}/scBio_{{ version }}.tar.gz
```
Comment (grst) [inline recipes/r-scbio/meta.yaml:9]: This URL is probably more stable as it is automatically generated by github: 
```
https://github.com/amitfrish/scBio/archive/refs/tags/{{ version }}.tar.gz
```

The other archive is manually generated and uploaded by the author.

File: recipes/r-scbio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  sha256: 06854bc33b1eac6c600cd81cebd23ab40cf451079f40e6d56f3cbbdf08dc8d78
+
+build:
+  number: 4
```
Comment (grst) [inline recipes/r-scbio/meta.yaml:13]: build number should be 0 initially

File: recipes/r-scbio/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+test:
+  commands:
+    - $R -e "library('scBio')"
+
+about:
+  home: https://github.com/amitfrish/scBio
+  license: GPL-2
+  summary: "Cellular population mapping (CPM) a deconvolution algorithm in which single-cell genomics is required in only one or a few samples, where in other samples of the same tissue, only bulk genomics is measured and the underlying fine resolution cellular heterogeneity is inferred."
+  license_family: other
```
Comment (grst) [inline recipes/r-scbio/meta.yaml:48]: license family should be `GPL` to be in agreement with `license`

**Review by grst — CHANGES_REQUESTED** [review]

**Comment by PelzKo** [comment]
@BiocondaBot please add label

**Review by grst — APPROVED** [review]

**Comment by druvus** [comment]
@BiocondaBot please merge

**Pull request merged**